### PR TITLE
Feature/semaphore

### PR DESCRIPTION
**Pull request by marcomancuso**

### CODE REVIEW

File: spec/chef_commons_semaphore_spec.rb [hunk 1]
```diff
@@ -130,22 +130,6 @@
...
-              "The specified bucket does not exist",
-              "The specified bucket does not exist"
-              )
-            }
-        }
-
-        expect(dummy_instancesemaphore.new.stop(@mynode)).to eq(true)
-      end
-    end
-
```
Comment (enzor) [inline spec/chef_commons_semaphore_spec.rb]: why did you remove it?

Comment (marcomancuso) [inline spec/chef_commons_semaphore_spec.rb]: as the commit says: "removing duplicated test", it was twice there
